apps/frontend/src/libs/components/carousel/libs/enums/carausel-config.enum.ts

```diff
@@ -0,0 +1,18 @@
+const carouselConfig = {
```

Comment:
should be in constants, not enum. Enums are not just objects; they are used to enumerate possible values for a specific constant, not to keep a set of constants and values
```suggestion
const CAROUSEL_CONFIG = {
```